package.json

```diff
@@ -1,9 +1,13 @@
 {
+  "scripts": {
+    "format": "prettier --write \"**/*.{js,json,ts,tsx}\""
```

Comment:
This include list misses other formats such as `.md` and `.yml`. Those files should be linted too.

https://github.com/typescript-eslint/typescript-eslint/issues/5060 has a good suggestion on what to use instead. Does that work?

(I don't know that this repo needs a `.prettierignore`, so `--ignore-path` might not be needed here -- unless it's formatting `node_modules/...` by default?)

Reply:
Yea I didn't want to include yml or md. I was feeling it might affect the workflows, I am not sure but I think yml must have an indentation or two? Also md files have a specific indentation size right? Or it doesn't matter ?🤔

Reply:
Neither file type cares about Prettier-induced whitespace changes. It doesn't matter. 

![Elmo raising hands above burning fire on a black background](https://user-images.githubusercontent.com/3335181/170190447-2543bd58-7039-4281-b0df-3306a8cbaa03.gif)

Reply:
Awesomeeee